Cargo.lock

```diff
@@ -1,3 +1,5 @@
+# This file is automatically @generated by Cargo.
```

Comment:
lol

Reply:
I included it because of [this](https://doc.rust-lang.org/cargo/faq.html#why-do-binaries-have-cargolock-in-version-control-but-not-libraries). I may have misinterpreted however, as artifact does contain libraries and binaries as far as I can tell.